.gitignore

```diff
@@ -7,3 +7,6 @@ out/
 
 # Ignores development broadcast logs (generated while interacting with contracts)
 broadcast/
+
+# Build artifacts
+**/artifacts
```

Comment:
Just out of curious, what tool produces this folder? Is it sage?

Reply:
I believe it's the solidity build artifacts when I use the Remix IDE.